src/olympia/files/models.py

```diff
@@ -354,6 +358,23 @@ def webext_permissions_list(self):
         except WebextPermission.DoesNotExist:
             return []
 
+    @cached_property
+    def optional_permissions_list(self):
```

Comment:
Could this be renamed `webext_optional_permissions` or simply `optional_permissions`?

Reply:
It seemed unnecessary and overly long to include `webext` in this property name, but I kept the `_list` suffix to keep it consistent with the `webext_permissions_list` property, which is why it ended up as `optional_permissions_list`. 

I could change it to just `optional_permissions`, which is in fact what I would probably have called it in a vacuum, but I still see some value in maintaining the consistency with `webext_permissions_list`. Therefore, my inclination is to just keep it as is, unless you feel strongly about it.

Reply:
The `webext` prefix is useful to know that this is coming from a related model I think. I'd go for `optional_permissions` then because, if the `webext` prefix is not needed, then the `_list` is definitely useless.

Reply:
FWIW I think consistency is important too - I don't particularly care about keeping the `webext_` prefix or the `_list` suffix, but whatever we choose we should do the same on the other property, IMHO.

Reply:
In this case I think just having `permissions` and `optional_permissions` is the best way to go. I'll make that change.